src/include/btmem.h

```diff
@@ -1140,7 +1143,7 @@ struct __wt_update {
  * WT_UPDATE_SIZE is the expected structure size excluding the payload data --
  * we verify the build to ensure the compiler hasn't inserted padding.
  */
-#define	WT_UPDATE_SIZE	38
+#define	WT_UPDATE_SIZE	39
```

Comment:
I think we might get into trouble here because a boolean isn't a fixed size, and so it might be padded? Should we make that a `uint8_t`?

Reply:
I did think about bool not having a standard defined size, thought this being a throw away code it should be fine. I decided to make in uint8_t